README.md

```diff
@@ -26,9 +26,26 @@ To build .NET Core 3.1 from source, pick a specific Git tag from this repo with
 
 ## Source-build goals
 
-The key goal of source-build is to satisfy the official packaging rules of commonly used Linux distributions, such as [Fedora](https://fedoraproject.org/wiki/Packaging:Guidelines) and [Debian](https://www.debian.org/doc/manuals/maint-guide/build.en.html). Many Linux distributions have similar rules. These rules tend to have two main principles: consistent reproducibility, and source code for everything.
+There are two primary goals of the source-build effort:
+
+1. Increase .NET adaption by focusing on making .NET available everywhere
+
+   If .NET was available everywhere - including Linux distributions and package managers like Homebrew - as a first class citizen it, would make .NET a more attractive option for developers who might otherwise look at other languages or runtimes. Users would be more likely to start using and keep using .NET if .NET is available on their development and release platforms of choice.
+
+   To achieve this, we try to make it easier for community and partner maintainers to build and release .NET for their platforms. We need to make sure source-build satisfies the official packaging rules of commonly used Linux distributions, such as [Fedora](https://fedoraproject.org/wiki/Packaging:Guidelines) and [Debian](https://www.debian.org/doc/manuals/maint-guide/build.en.html). Many Linux distributions have similar rules. These rules tend to have three main principles:
+
+   - Limited or no network access
+   - Consistent reproducibility
+   - Source code for everything.
+
+2. Make maintenance of the .NET product easier
```

Comment:
I feel like this has never been an actual goal of source-build.  To-date, I think source-build has actually made maintenance of the product more difficult.

Things are changing however with the unified build project however.  I think the unified build project will address the areas of improving serviceability and supporting a true upstream/downstream model.

Reply:
@mmitche and @premun are going to be making a PR shortly for the unified build documentation.  Once that is in place I think we should link to it as it makes sense.

Reply:
https://github.com/dotnet/arcade/pull/11439

Reply:
https://github.com/dotnet/arcade/blob/main/Documentation/UnifiedBuild/Overview.md is the link to the landing page.

Reply:
Done. Thanks!